### DISCUSSION TITLE
Default date/time offset based on publish date

### DISCUSSION THREAD

**Discussion by jhanten** [publishpress/publishpress-future#531] [Feature Requests]
Please consider having an option to base the Default date/time offset to be based on the publish date instead of the current date (like for a new post). I believe this would be a better user experience for people starting to use your plugin on an existing site with existing content. Your current method makes sense for new content, but is annoying for old content.

For example if I set the default to expire content to +2 years and I try to quick edit an existing post to add Future Action it defaults to 2 years from the day I am editing. The behavior I expected was to show me 2 years from the date the post was original posted.

Another benefit to this idea might be in even better support for Bulk Editing because if the field could be set to just show the offset and just be updated it would make the formula work for each post being editing based on publish date.

I hope my example makes sense and thank you for your consideration.

**Comment by stevejburge** [comment]
This is now available in Future Pro
